From eba5ea94dcbeb6936aef45177e82ae18d62733f1 Mon Sep 17 00:00:00 2001
From: James Su <james.su@gmail.com>
Date: Wed, 12 May 2010 15:38:42 -0700
Subject: [PATCH 03/10] Change default values of some config.

This CL changes the default value of use_global_engine, enable_by_default and use_sys_layout to TRUE. And also removes the default trigger key.

BUG=none
TEST=none

Review URL: http://codereview.chromium.org/2008016
---
 bus/ibusimpl.c |    9 ++++++---
 1 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/bus/ibusimpl.c b/bus/ibusimpl.c
index 51acd12..b7af0f5 100644
--- a/bus/ibusimpl.c
+++ b/bus/ibusimpl.c
@@ -189,12 +189,15 @@ bus_ibus_impl_set_trigger (BusIBusImpl *ibus,
 {
     GQuark hotkey = g_quark_from_static_string ("trigger");
     bus_ibus_impl_set_hotkey (ibus, hotkey, value);
+#if 0
+    /* Chromium-OS: We don't use trigger key. */
     if (value == NULL) {
         ibus_hotkey_profile_add_hotkey (ibus->hotkey_profile,
                                         IBUS_space,
                                         IBUS_CONTROL_MASK,
                                         hotkey);
     }
+#endif
 }
 
 static void
@@ -617,10 +620,10 @@ bus_ibus_impl_init (BusIBusImpl *ibus)
     ibus->hotkey_profile = ibus_hotkey_profile_new ();
     ibus->keymap = ibus_keymap_get ("us");
 
-    ibus->use_sys_layout = FALSE;
+    ibus->use_sys_layout = TRUE;
     ibus->embed_preedit_text = TRUE;
-    ibus->enable_by_default = FALSE;
-    ibus->use_global_engine = FALSE;
+    ibus->enable_by_default = TRUE;
+    ibus->use_global_engine = TRUE;
     ibus->global_engine = NULL;
     ibus->global_previous_engine_name = NULL;
 
-- 
1.7.1

